### PR TITLE
:bug: Fixes a small bug in the remark-lint syntax

### PR DESCRIPTION
**Pull request by rickstaa**
This pull request fixes a small bug which prevented the remark-lint action to work with the review dog github-pr-review action. The file path is now first ignored and then pushed onto the stack.

### CODE REVIEW

**Comment by rickstaa** [comment]
I'm unsure why the travis action fails. I think it has to do with a upstream bug as the error it gives:

```bash
reviewdog/reviewdog info checking GitHub for latest tag
reviewdog/reviewdog crit unable to find '' - use 'latest' or see https://github.com/reviewdog/reviewdog/releases for details
```

Has no relationship to the files I changed.

File: .gitignore [hunk 1]
```diff
@@ -1 +1,5 @@
-.drone.sec.yaml
+# Folders to ignore
+.vscode/
+
+# Files to ignore
+.drone.sec.yaml
```
Comment (mgrachev) [inline .gitignore:5]: Please add a new line.

**Review by mgrachev — COMMENTED** [review]
@rickstaa Thank you for your contribution! 👍

I have left a comment. Please take a look at it 👀

**Comment by mgrachev** [comment]
Please remove the `errorformat` file from your PR.

File: .gitignore (hunk 1 above)
Comment (rickstaa) [reply .gitignore]: Great thanks a lot for pointing that out!

**Review by mgrachev — APPROVED** [review]

**Comment by mgrachev** [comment]
@rickstaa Thank you 💪

**Pull request merged**